recipe-server/client/control_new/tests/components/common/test_EnvAlert.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import EnvAlert from 'control_new/components/common/EnvAlert';
+
+describe('<EnvAlert>', () => {
+  it('should work', () => {
+    const wrapper = () => shallow(<EnvAlert />);
+
+    expect(wrapper).not.toThrow();
+  });
+
+  describe('findPartialString', () => {
+    const testStrings = ['abc', 'def'];
+
+    it('should find a piece of a string amongst an array of strings', () => {
```

Comment:
This test should probably use representative urls, instead of filler data. That way it's clearer if it's testing the right thing.